sparrowpy/classes/RadiosityFast.py

```diff
@@ -429,13 +437,21 @@ def init_source_energy(
         brdf = np.array(self._brdf)
         brdf_index = self._brdf_index
         patches_center = self.patches_center
+        source_vis = geometry._check_point2patch_visibility(
+                                        eval_point=source_position,
+                                        patches_center=patches_center,
+                                        surf_points=self.walls_points,
+                                        surf_normal=self.walls_normal)
+
```

Comment:
```suggestion
    self._source_visibility = source_visibility
```
I would still save the visbility matrix from source, but not from receiver